README.md

```diff
@@ -24,6 +24,27 @@ them are as follows:
     openldap_serverlocation: Portland
     openldap_serverorganization: IT
 
+    # Create users
+    openldap_enable_user_creation: true
+    openldap_server_users:
+      staff:
+        william:
+          password: 26April1564
+          fullname: William Shakespeare
+          firstname: William
```

Comment:
I did consider that, although personally I've never liked the attribute name 'givenname' as it seems meaningless to most people.  Maybe it's just me, so I'll change it to givenname.